### PR TITLE
Adding some more protocols in s3

### LINKED ISSUE #40: Adding some more protocols in s3
ekr had the following comment:

    You should probably list DTLS, SCTP, and SDP in this section. It's
    not like we haven't decided we need them.

### PR DESCRIPTION
**Pull request by seanturner**
Closes #40.

### CODE REVIEW

File: rtcweb-overview.xml [hunk 1]
```diff
@@ -457,12 +457,12 @@
 
       <t><list style="symbols">
           <t>Data transport: TCP, UDP and the means to securely set up
-          connections between entities, as well as the functions for deciding
-          when to send data: Congestion management, bandwidth estimation and
-          so on.</t>
+          connections between entities, as well as the functions for
+          deciding when to send data: congestion management, bandwidth
+          estimation and so on.</t>
```
Comment (ekr) [inline rtcweb-overview.xml]: , after estimation

File: rtcweb-overview.xml [hunk 2]
```diff
@@ -457,12 +457,12 @@
...
 
-          <t>Data framing: RTP, SCTP and other data formats that serve as
-          containers, and their functions for data confidentiality and
+          <t>Data framing: RTP, SCTP, DTLS and other data formats that serve
```
Comment (ekr) [inline rtcweb-overview.xml]: I suspect that you want a comma after DTLS, but @juberti is your man for that.

**Review by ekr — APPROVED** [review]
LGTM with nits

File: rtcweb-overview.xml (hunk 2 above)
Comment (seanturner) [reply rtcweb-overview.xml]: Yeah I'll put the comma in.  I'd rather not get embroiled in an oxford comma debate.

**Review by martinthomson — APPROVED** [review]

**Pull request merged**